$OpenBSD: patch-src_player_c,v 1.7 2011/07/10 11:01:05 dcoppa Exp $

Add missing include

Our faad port uses a patched neaacdec.h header

--- src/player.c.orig	Sat Jul  9 10:40:54 2011
+++ src/player.c	Sun Jul 10 12:40:48 2011
@@ -28,6 +28,7 @@ THE SOFTWARE.
 #include <math.h>
 #include <stdint.h>
 #include <limits.h>
+#include <netinet/in.h>
 #include <arpa/inet.h>
 
 #include "player.h"
@@ -186,7 +187,7 @@ static WaitressCbReturn_t BarPlayerAACCb (void *ptr, s
 					/* +1+4 needs to be replaced by <something>! */
 					player->bufferRead += 1+4;
 					char err = NeAACDecInit2 (player->aacHandle, player->buffer +
-							player->bufferRead, 5, &player->samplerate,
+							player->bufferRead, 5, (void *)&player->samplerate,
 							&player->channels);
 					player->bufferRead += 5;
 					if (err != 0) {
